### DISCUSSION TITLE
Unitary from TensorNetwork object?

### DISCUSSION THREAD

**Discussion by inajetovic** [jcmgray/quimb#236] [Q&A]
Hi all,
I'm super new to quimb, but I'm really interested into that. Is there a method to get the unitary of a TensorNetwork object?
Probably I'm missing something.
Thanks in advance!

**Comment by jcmgray** [comment]
Hi @inajetovic, tensor networks often represent unitaries, if you mean the 'dense' / 'matrix' version of a unitary then (as long as it will fit in memory) forming it is usually simply a matter of contracting the TN. Happy to give more information if you have a specific context.
